react/features/conference/components/native/Conference.js

```diff
@@ -164,7 +162,9 @@ class Conference extends AbstractConference<Props, *> {
         const { _fullscreenEnabled, _showLobby } = this.props;
 
         if (_showLobby) {
-            return <LobbyScreen />;
+            ConferenceNavigationContainerRef.navigate('Lobby');
```

Comment:
This feels a bit weird as the place to navigate to the lobby no? Basically we'd be navigating to the conference over and over on every prop change. What is usually the way to accomplish this?

Reply:
So I added this because after entering the password to join or getting acces, the lobby screen didn't close and I couldn't navigate to the conference room

Reply:
Sight. I wonder if it doesn't make more sense to move it to componentDidUpdate. We can explicitly check if the value of that prop has changed and navigate accordingly. WDYT?

Reply:
Will do.

Reply:
Moved the check to componentDidUpdate.